namespace_registry/registry.go

```diff
@@ -763,13 +802,22 @@ func getOpenIDConfiguration(c *gin.Context) {
 */
 
 func RegisterNamespaceRegistry(router *gin.RouterGroup) {
-	registry := router.Group("/api/v1.0/registry")
+	registry1 := router.Group("/api/v1.0/registry")
```

Comment:
Can we make these a little more clear that we're versioning? Maybe something like `v1registry` and `v2registry`?